addons/libs/images.lua

```diff
@@ -387,17 +398,25 @@ windower.register_event('mouse', function(type, x, y, delta, blocked)
     if type == 0 then
         if dragged then
             dragged.image:pos(x - dragged.x, y - dragged.y)
+            call_events(dragged.image, 'drag')
+            dragged.image:last_pos(dragged.image:pos())
             return true
         end
 
     -- Mouse left click
     elseif type == 1 then
         for _, t in pairs(saved_images) do
```

Comment:
**SEPARATE PR:** If we wanted to, this could be changed to a backwards iteration that, on the first match, does all the existing stuff but also breaks out. This may be useful for addons that have stacked images, to make the click event only fire for the addon's top-most image in the mouse's x/y region.